lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1805,7 +1807,7 @@ export class UserAgentApplication {
      * @param state
      * @return {@link AuthResponse} AuthResponse
      */
-    protected getCachedTokenInternal(scopes : Array<string> , account: Account, state: string): AuthResponse {
+    protected getCachedTokenInternal(scopes : Array<string> , account: Account, state: string, correlationId?: string): AuthResponse {
```

Comment:
Do we use the correlation id when looking up the cached token?

Reply:
No, as tokens can be used for multiple requests.

Reply:
I believe it is used in creating the serverRequestParameters, which is constructed in this function. However, this particular function (getCachedTokenInternal) is only used by the Angular wrapper, so it may not be necessary.

Reply:
So should we not pass it in here?

Reply:
I kept this intact for now to keep the interface the same, ideally I think we should rewrite this function and `getCachedToken` to not have dependency on the `ServerRequestParameters`, as they do not need a NW call. But it is too much code change that is irrelevant to the current problem we are solving.

Reply:
cc @jasonnutter, please let me know if we co-ordinate on refactoring this piece of code in `msal-core` as you start working on angular.

Reply:
For 1.0 of Angular, I'm going to leave this code (and the other Angular-specific code). We'll remove it for msal-core v2.

Reply:
ok resolving